mix.exs

```diff
@@ -54,15 +54,16 @@ defmodule Membrane.MP3.Lame.Mixfile do
 
   defp deps do
     [
-      {:membrane_core, "~> 0.8.0"},
-      {:membrane_caps_audio_raw, "~> 0.5.0"},
+      {:membrane_core, "~> 0.9.0"},
+      {:membrane_raw_audio_format, "~> 0.8.0"},
       {:membrane_caps_audio_mpeg, "~> 0.2.0"},
-      {:membrane_common_c, "~> 0.10.0"},
+      {:membrane_common_c, "~> 0.11.0"},
       {:bunch, "~> 1.0"},
       {:unifex, "~> 0.7.0"},
       {:ex_doc, "~> 0.22.2", only: :dev, runtime: false},
-      {:espec, "~> 1.7", only: :test},
```

Comment:
You should not remove espec without rewriting the tests under `spec` dir to  ExUnit

Reply:
Ah, I only checked `test` folder